src/Rubric/RubCharacterBlockScanner.class.st

```diff
@@ -91,7 +91,7 @@ RubCharacterBlockScanner >> cr [
 	CharacterBlocks, sometimes including the carriage return (cursor is at 
 	the end) and sometimes not (cursor is in the middle of the text)."
 
-	((characterIndex ~= nil
+	((characterIndex notNil
```

Comment:
I don't think that difference is really measurable nowadays (more over, the frequency of cr is probably about 1/20 in typical Smalltalk code), but the purpose of this kind of optimization was to make the full Smalltalk character rendering inner loop feasible on slow machines.

In doubt, one could use `ifNotNil:` to ensure same level of optimization.

Or we could profile performance of rendering as part of non regression tests...